script/build_libcec

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+# Sets up and builds libcec to be used with Home Assistant.
+# Dependencies that need to be installed:
+# apt-get install cmake libudev-dev libxrandr-dev python-dev swig
+
+# Stop on errors
+set -e
+
+cd "$(dirname "$0")/.."
+mkdir -p build && cd build
+
+if [ ! -d libcec ]; then
+  git clone --branch release --depth 1 https://github.com/Pulse-Eight/libcec.git
+fi
+
+cd libcec
+git checkout release
+git pull
+git submodule update --init src/platform
+
+# Build libcec platform libs
+(
+    mkdir -p src/platform/build
+    cd src/platform/build
+    cmake ..
+    make
+    make install
+)
+
+# Fix upstream install hardcoded Debian path.
+sed -i \
+    -e '/DESTINATION/s/dist-packages/site-packages/' \
+    src/libcec/cmake/CheckPlatformSupport.cmake
+
+# Build libcec
+(
+    mkdir -p build && cd build
+
+    cmake \
+        -DPYTHON_LIBRARY="/usr/local/lib/libpython3.5m.so" \
```

Comment:
Could we make this depend on the current running version of Python? Because I was planning on updating our docker image to use Python 3.6 but I don't want such updates to be blocked by installing a dependency.